### ISSUE TITLE
 a question about model regularization

### ISSUE THREAD

**Issue by Yinyiyang0504** [ChristianKaltenecker/HINNPerf#1]
"def __add_loss_op(self):
        """Defines self.loss"""
        l2_loss = tf.losses.get_regularization_loss()
        self.loss = l2_loss + tf.losses.mean_squared_error(self.Y, self.output)"
the code of model regularization is L2_regularization, but reading the paper, it seems like using L1_regularization?

**Comment by ChristianKaltenecker** [comment]
Hey,
If I remember correctly, you are right.
As stated in the README of this repository:
This is not the original HINNPerf tool. Please be adviced that the code of HINNPerf has changed in this repository to prepare HINNPerf for further comparison to other ML tools!

One of those changes was the L2 regularization.
We chose to use go with the L2 regularization for the sake of comparing it to other tools that also used L2 regularization.
Feel free to fork this project and change this as desired.
